e2e/tests/components/toolbar-search/toolbar-search.e2e-spec.ts

```diff
@@ -31,6 +31,21 @@ describe('Toolbar Search', () => {
 
     });
 
+    it('should display the name of the selected country', async () => {
+
+        // selecting country with clicking
+        await page.clickOnDropdown();
+        expect(await page.confirmDropdownIsExpanded()).toBeFalsy();
+
+        expect(await imageCompare('toolbar-search-typeahead-open')).toEqual(0);
+
+        // Enter search text
+        await page.leftInput.sendKeys('United States');
+
+        expect(await page.getSelectedLocationText()).toBe('United States');
```

Comment:
I think this can be removed, we don't need to test selection functionality as this would be covered by typeahead tests